sdk/core/azure-core/inc/ApiViewSettings.json

```
@@ -0,0 +1,21 @@
+{
+  "sourceFilesToProcess": null,
+  "sourceFilesToSkip": [
+    "azure/core/internal/json/json.hpp",
+    "azure/core/internal/json/json_optional.hpp"
+  ],
+  "additionalIncludeDirectories": [
+  ],
+  "additionalCompilerSwitches": [
+    "-DBUILD_CURL_HTTP_TRANSPORT_ADAPTER",
+    "-DBUILD_TRANSPORT_WINHTTP_ADAPTER",
+    "-DCURL_STATICLIB"
+  ],
+  "allowInternal": true,
+  "includeDetail": false,
+  "includePrivate": false,
+  "filterNamespace": "Azure::",
+  "reviewName": "Azure Core Review",
```

We should be consistent with the naming across all the API reviews.

I see three different variants of `reviewName` in this PR. Is it:
- `<service/package name> API Review`
- `<service/package name> Review`
- `<service/package name>`

The first one makes sense to me, but as long as they are all consistent, any is fine.

```suggestion
"reviewName": "Azure Core API Review",
```

---

FWIW, my expectation is that this field will never be used. The CI pipeline will override the reviewName field with somethign which is instanced on a per-changelist basis. This field is only there because the ApiView web site wants it and we don't have the CI pipeline in place.